packages/eslint-plugin/src/rules/return-await.ts

```diff
@@ -113,24 +106,17 @@ export default util.createRule({
 
     function insertAwait(
       fixer: TSESLint.RuleFixer,
-      node: TSESTree.ReturnStatement | TSESTree.ArrowFunctionExpression,
+      node: TSESTree.Expression,
     ): TSESLint.RuleFix | null {
-      const targetNode =
-        node.type === AST_NODE_TYPES.ReturnStatement
-          ? node.argument
-          : node.body;
       // There should always be a target node; but let's be safe.
-      /* istanbul ignore if */ if (!targetNode) {
+      /* istanbul ignore if */ if (!node) {
         return null;
       }
```

Comment:
you don't need this if any more, because the argument is never `undefined`

Reply:
@bradzacher Sure!